quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -163,7 +164,7 @@ type instance Realized (WriterT w m) a = Realized m a
 type instance Realized Identity a = a
 
 newtype PostconditionM m a = PostconditionM {runPost :: WriterT (Endo Property, Endo Property) m a}
-  deriving (Functor, Applicative, Monad, MonadTrans)
```

Comment:
What's the idea behind this change? A nice part of having monadic postconditions is to be able to perform e.g. lookup operations to compare the result of an action to some part of the underlying system.

Reply:
And yes, if the whole idea here is that it's not necessary to have this derive because it follows from something else and that causes some issue with compilation somewhere then go right ahead :)